sw/device/info_sections.ld

```diff
@@ -0,0 +1,81 @@
+/* Copyright lowRISC contributors. */
+/* Licensed under the Apache License, Version 2.0, see LICENSE for details. */
+/* SPDX-License-Identifier: Apache-2.0 */
+
+/**
+ * Partial Linker Script for OpenTitan Device Executables
+ *
+ * This Linker Script controls all the sections that we need in the ELF file for
+ * a device executable, but which are not mapped into memory or otherwise
+ * allocated on the device itself.
+ *
+ * These contain amongst other things:
+ * - debugging information needed by DV logging,
+ * - DWARF Sections
+ * - RISC-V Attributes sections
+ *
+ * This is also where we specify the sections to discard, because it is a common
+ * file between all our other linker scripts.
+ */
+
+/*
+ * The following sections are used by DV to implement logging in an
+ * alternate way, which enables simulation speed up by completely avoiding
+ * any string format processing or even the actual transmission of log data
+ * to a real peripheral.
+ *
+ * These sections are marked as dummy so that they can still be extracted
+ * using readelf or similar utilities. As such, the content in these sections
+ * is not relevant for the actual SW code and can be safely discarded.
+ */
+
+/**
+ * The following section contains log fields constructed from the logs using
+ * the log_fields_t struct defined in sw/device/lib/runtime/log.h. The size of
+ * each log field is fixed - 20 bytes, which is used as the delimiter.
+ */
+.logs.fields 0x0 (INFO): {
+  *(.logs.fields)
+}
+
+/**
+ * End DV Sections
+ */
```

Comment:
Is this comment needed? (I don't think so). If you *do* think so, I think there should probably also be a "start DV sections" comment.

Reply:
There is a "Start DV sections" comment - it says "The following sections are used by DV…".

Reply:
Oh, cool! Sorry, I think I must have missed it.